common/include/pcl/common/point_tests.h

```diff
@@ -105,19 +105,22 @@ namespace pcl
   }
 
   // generic fallback cases
-  template <typename PointT, typename = void, typename = void > constexpr inline bool
+  template< class... >
+  using void_t = void;
```

Comment:
We have this in `point_traits` already:

https://github.com/PointCloudLibrary/pcl/blob/96b6f4ef177c5ddef8f76492752d55577edb21e6/common/include/pcl/point_traits.h#L341

Reply:
Oh, didn't notice but that line shouldn't have gone in. I'm not using it at all, was just experimenting